### ISSUE TITLE
Little bug when using AxisAlignedWire

### ISSUE THREAD

**Issue by crane-may** [zakarumych/egui-snarl#30]
I've found that with AxisAlignedWire if `to` appears to the left of `from`, the line may disappear when the screen moves.

Then I found a change in one line of code that solved the problem.

src/ui/wire.rs line 696:

```rust
fn draw_axis_aligned(
    ui: &Ui,
    shapes: &mut Vec<Shape>,
    corner_radius: f32,
    frame_size: f32,
    from: Pos2,
    to: Pos2,
    stroke: Stroke,
) {
    let aabb = Rect::from_two_pos(from, to);  // original code:  let aabb = Rect::from_min_max(from, to);

    if !ui.is_rect_visible(aabb) {
        return;
    }

    
...
```

Btw, I love this toolkit.

**Comment by zakarumych** [comment]
Thanks.
Fixed in 983b9cc7cc7bd34e319d3091e95d98c5c65ec39a

**Comment by zakarumych** [comment]
Released in [0.4.4](https://crates.io/crates/egui-snarl/0.4.4)
